guides/routing/README.md

```diff
@@ -0,0 +1,5 @@
+# Routing, Middleware and Context
+
+{% import "../../macros.html" as macros %}
+
+{{ macros.tocGenerator(page.title, summary.parts[0].articles[2].articles[8].articles) }}
```

Comment:
`.articles[2]` has to be changed to `.articles[3]` since the `starcounter` section was added.